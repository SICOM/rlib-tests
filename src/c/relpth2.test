#
# Same test as relpth1 but it uses PNG images
# RLIB 2.0.x fails to add the PNG images to the PDF output.
#
COMMAND=./relpth2
FORMATS="pdf xml txt csv html"
EXPECTED_pdf=
EXPECTED_xml="1 2"
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html="1 2"
