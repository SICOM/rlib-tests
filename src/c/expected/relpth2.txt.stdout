Testing relative path
Below there should be a red A and a blue B


Below there should be a green C and a red D


